src/main/features/directions-questionnaire/index.ts

```diff
@@ -0,0 +1,39 @@
+import * as express from 'express'
+import * as path from 'path'
+import { OAuthHelper } from 'idam/oAuthHelper'
+import { AuthorizationMiddleware } from 'idam/authorizationMiddleware'
+import { ClaimMiddleware } from 'claims/claimMiddleware'
+import { CountyCourtJudgmentRequestedGuard } from 'response/guards/countyCourtJudgmentRequestedGuard'
+import { DraftMiddleware } from '@hmcts/cmc-draft-store-middleware'
+import { DraftService } from 'services/draftService'
+import { DirectionsQuestionnaireDraft } from 'directions-questionnaire/draft/DirectionsQuestionnaireDraft'
+import { RouterFinder } from 'shared/router/routerFinder'
+
+function requestHandler (): express.RequestHandler {
+  function accessDeniedCallback (req: express.Request, res: express.Response): void {
+    res.redirect(OAuthHelper.forLogin(req, res))
+  }
+
+  const requiredRoles = ['citizen']
+  const unprotectedPaths = []
+  return AuthorizationMiddleware.requestHandler(requiredRoles, accessDeniedCallback, unprotectedPaths)
+}
+
+export class DirectionsQuestionnaireFeature {
+  enableFor (app: express.Express) {
+    const allDQs = '/case/*/directions-questionnaire/*'
+    app.all(allDQs, requestHandler())
+    app.all(allDQs, ClaimMiddleware.retrieveByExternalId)
+    app.all(allDQs, CountyCourtJudgmentRequestedGuard.requestHandler)
+    app.all(allDQs,
```

Comment:
Quick thought, we should probably have a guard here to check that the claim has the new 'directions-questionnaire' feature on it